src/openstudio_lib/GeometryEditorView.cpp

```diff
@@ -196,6 +208,13 @@ FloorspaceEditor::FloorspaceEditor(const openstudio::path& floorplanPath, bool i
       model::FloorplanJSForwardTranslator ft;
       m_floorplan = ft.updateFloorplanJS(*m_floorplan, m_model, true);
 
+      // update north axis from model, floorspace js northAxis is opposite of EnergyPlus's
+      if (building) {
+        m_floorplan->setNorthAxis(-building->northAxis());
+      }
```

Comment:
Hum...

Reply:
What was the concern, the difference in north angle convention?

Reply:
Yeah.